### PR TITLE
Various shared utils for rubrics

### PR DESCRIPTION
**Pull request by bethanyaconnor**
A miscellaneous PR of various utilities / constants for rubrics:
- A shared constant for the evidence levels we support, with validation added to `LearningGoalEvidenceLevel`
- A function to convert those evidence levels to translatable strings
- React shapes for the rubric models

I pulled these out as it would help me parallelize my work more efficiently and I think some of these are also relevant to Kaitie's current work.

### CODE REVIEW

File: apps/src/templates/rubrics/rubricShapes.jsx
Comment (kobryan0619) [inline apps/src/templates/rubrics/rubricShapes.jsx:17]: Should this also have the lesson / level associated with this rubric?

Comment (bethanyaconnor) [reply apps/src/templates/rubrics/rubricShapes.jsx:17]: hm, I'm not sure. I'm definitely thinking of this on the teacher side where we might not -- we already know the lesson and level of the page in those cases. Would it be helpful in the curriculum writer case? We can add them into this shape and not use them for the teacher side

File: dashboard/test/fixtures/test-serialize-seeding-json.script_json
Comment (kobryan0619) [inline dashboard/test/fixtures/test-serialize-seeding-json.script_json:1]: This is my first time reading Ruby tests, I was surprised to see this seeding script only went through understanding levels 0 and 1.  I would kind of expect all the levels to be tested or just one of them.

**Review by kobryan0619 — APPROVED** [review]
Thanks for putting together the shapes and the shared constants.  I had some questions that were perhaps more related to me being newer to the ways of ruby than anything else.  I am approving these but also understand if you want a second opinion/approval!

File: dashboard/test/fixtures/test-serialize-seeding-json.script_json
Comment (bethanyaconnor) [reply dashboard/test/fixtures/test-serialize-seeding-json.script_json:1]: Fair point. It's creating two fairly arbitrarily from [this line](https://github.com/code-dot-org/code-dot-org/blob/d50dd41196bca92d8c7e2c727aa0452c85a52579/dashboard/test/lib/services/script_seed_test.rb#L1427). I think my thinking was that will almost always be multiple evidence levels per learning goal, but, honestly, I could've tested with just one.

File: apps/src/templates/rubrics/rubricShapes.jsx
Comment (kobryan0619) [reply apps/src/templates/rubrics/rubricShapes.jsx:17]: I think this was a misconception on my part.  I see now that `PropTypes.shape` allows other properties to be included with the object (vs `PropTypes.exact`).  With that, I think we are good here for now!

Comment (bethanyaconnor) [reply apps/src/templates/rubrics/rubricShapes.jsx:17]: Ooo I didn't know about `PropTypes.exact`! Maybe something for us to try once we get a little farther in development -- I would try it now but I'm having a bit of trouble finding up to date docs on it :/

**Pull request merged**
